File: 프로그래머스/0/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.py
def solution(arr):
    answer = []
    for num in arr:
        n = num
        if num >= 50 and num % 2 == 0:
            n =num/2
        elif num < 50 and num % 2 == 1:
            n = num*2
        answer.append(n)
    return answer

File: 프로그래머스/0/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr) {
        List<Integer> answer = new ArrayList<Integer>();
        for (int num : arr) {
            if (num >= 50 && (num % 2 == 0)) {
                answer.add(num / 2);
            } else if (num < 50 && num % 2 == 1){
                answer.add(num * 2);
            } else {
                answer.add(num);
            }
        }
        return answer.stream().mapToInt(i -> i).toArray();
    }
}